### PR TITLE
Fix progress report (#12), emit ZeroEQ progress event

### PR DESCRIPTION
**Pull request by eile**

### CODE REVIEW

**Comment by eile** [comment]
@rdumusc @tribal-tec FYI

**Comment by jafyvilla** [comment]
Maybe you can squash the last commit (Add voxelize_batch app), as I see it was just a conflict fix

File: doc/Changelog.md [hunk 1]
```diff
@@ -3,6 +3,8 @@ Changelog {#changelog}
 
 # git master {#master}
 
+* [#12](https://github.com/BlueBrain/Fivox/issue/12)
```
Comment (jafyvilla) [inline doc/Changelog.md]: No reference to the PR?

Comment (eile) [inline doc/Changelog.md]: Bug has ref to PR?

**Comment by eile** [comment]
Squashed the last commit.

**Comment by eile** [comment]
Retest this please.

**Comment by jafyvilla** [comment]
+1

On 04 Apr 2016, at 15:30, Stefan Eilemann <notifications@github.com<mailto:notifications@github.com>> wrote:


Retest this please.

**Pull request merged**